app/services/school_order_state_and_cap_update_service.rb

```diff
@@ -17,8 +19,16 @@ def update!
       allocation = update_cap!(cap[:device_type], cap[:cap])
 
       if notify_computacenter_of_cap_changes?
-        update_cap_on_computacenter!(allocation.id)
-        notify_computacenter_by_email(allocation.device_type, allocation.cap)
+        if FeatureFlag.active? :virtual_caps
+          # don't send updates as they will happen when the pool is updated and the caps adjusted
+          unless allocation.is_in_virtual_pool?
+            update_cap_on_computacenter!(allocation.id)
+            notify_computacenter_by_email(@school, allocation.device_type, allocation.cap)
+          end
+        else
+          update_cap_on_computacenter!(allocation.id)
+          notify_computacenter_by_email(@school, allocation.device_type, allocation.cap)
+        end
```

Comment:
Not sure... we don't want to do the `unless allocation.is_in_virtual_pool?` test until the FeatureFlag is set because at the moment we want the virtual pool to be transparent and not affect cap updates for allocations that are actually in a pool currently... if that makes sense?

Reply:
Makes sense - thanks.